src/components/BounceRulesActivity/index.jsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { CSVLink } from "react-csv";
+import "./index.scss";
+import Breadcrumb from "@sendgrid/ui-components/breadcrumb";
+import { StatefulTabs as Tabs, Tab } from "@sendgrid/ui-components/tabs";
+import Loader from "@sendgrid/ui-components/loader";
+import PropTypes from "prop-types";
+import { Redirect } from "react-router";
+import Header from "../Header";
+import { Row } from "../Row";
+import { Column } from "../Column";
+import RuleFilter from "./RuleFilter";
+import Pagination from "../Pagination";
+import EmptyRules from "./EmptyRules";
+import ActivityLogContainer from "./ActivityLogContainer";
+import { WriteSelectors } from "./selectors";
+import ActivityDetailsModal from "./ActivityDetailsModal";
+
+const BounceRulesContainer = ({
+  updateSearchToken,
+  updateSearchCategory,
+  removeFilter,
+  filteredActivityLog,
+  searchToken,
+  rulesToShow,
+  pagesToDisplay,
+  filterOptions,
+  addFilter,
+  invalidFilter,
+  handleModalClose,
+  isFetching,
+  logout,
+  handleActivityTabClicked,
+  handleBounceTabClicked,
+  isBounceRulesTab,
+  isActivityLogTab,
+  activityLog,
+  updatePageIndex,
+  handlePrevClicked,
+  handleNextClicked,
+  currentActivityPageIndex,
+  handleActivityClicked,
+  selectedActivity,
+  isActivityModalOpen,
+}) => {
+  const isActivityEmpty = activityLog.length <= 0;
+  const shouldShowActivityLogPagination =
+    isActivityLogTab && !isActivityEmpty && !isFetching;
+  const shouldShowActivityLogContainer = isActivityLogTab && !isActivityEmpty;
+  const shouldShowEmpty = isActivityEmpty && !isFetching;
+  return (
+    <React.Fragment>
+      {isBounceRulesTab && <Redirect push to="/bounce_rules" />}
+      <div {...WriteSelectors.page} className="container">
+        <Header logout={logout} />
+        <Row>
+          <Column width={6} offset={2}>
+            <Breadcrumb>
+              <a {...WriteSelectors.breadcrumb} href="/activity_log">
```

Comment:
I think for the links there's an actual Link component from react-router we can use but if anything this is okay to leave as is if it's not working with Link